SPECS/edk2/edk2.spec

```diff
@@ -67,7 +70,9 @@ Source2: openssl-rhel-%{OPENSSL_COMMIT}.tar.xz
 Source3: softfloat-%{softfloat_version}.tar.xz
 Source4: edk2-platforms-%{PLATFORMS_COMMIT}.tar.xz
 Source5: jansson-2.13.1.tar.bz2
-Source6: README.experimental
+Source6: hvloader-%{HVLOADER_COMMIT}.tar.gz
```

Comment:
not too opposed: I can see why you want to change the order of SOURCES but I would overthink it. To keep the change smaller and to keep the new hvloader sources adjacent to each other, I would not advise to shift README.experimental down